x-pack/plugins/siem/public/components/navigation/breadcrumbs/index.ts

```diff
@@ -42,6 +42,13 @@ export const rootBreadcrumbs: { [name: string]: Breadcrumb[] } = {
       href: getNetworkUrl(),
     },
   ],
+  timelines: [
+    ...siemRootBreadcrumb,
```

Comment:
I wouldn't prepend the word `siem` to anything. It's a product name. I would just name this `rootBreadCrumb`.

Reply:
The statement that immediately follows the declaration of the `siemRootBreadcrumb` variable (in the existing code) is the declaration of another variable named `rootBreadcrumbs`, which is very similar to your suggestion of `rootBreadCrumb`.

It appears that yesterday, you updated the line of code where `siemRootBreadcrumb` is declared. Neither of us are the original author, but you also approved the PR with the original `siemRootBreadcrumb` name ~ two weeks ago.

Perhaps in both cases, you were seeing the complete context of the file, and thus didn't make (or suggest) this change?

Reply:
It's fine to keep.